doc/authoring_command_modules/README.md

```diff
@@ -79,6 +63,10 @@ Command modules should have the following structure:
 `-- HISTORY.rst
 ```
 
+You could create these files manually, or [create a module with `azdev cli create`](https://azurecliprod.blob.core.windows.net/videos/04%20-%20AzdevCliCreate.mp4).
+
+You could also consider using [Code-gen tool](https://azurecliprod.blob.core.windows.net/videos/04%20-%20AzdevCliCreate.mp4) to generate code automatically.
```

Comment:
The link is not right.

Shall we move `Code-gen` above `azdev cli create` as it is more recommended now?

Reply:
good check!